### PR TITLE
Updates QA and support tooling/content

### PR DESCRIPTION
**Pull request by weierophinney**
This patch does the following:

- Ran `zf-maintainer rebase-doc-templates` over the repo, adding PR and issue templates and support and contributing documentation.
- Updates to zend-coding-standard, and applies it to the code base.
- Updates all file-level license docblocks.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,15 +49,38 @@
...
-        "cs": "phpcs",
+        "cs-check": "phpcs",
         "cs-fix": "phpcbf",
-        "test": "phpunit",
-        "test-coverage": "phpunit --coverage-clover clover.xml"
-    }
+        "test": "phpunit --colors=always",
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+    },
+    "homepage": "https://github.com/zendframework/ZendDeveloperTools",
```
Comment (webimpress) [inline composer.json]: we can remove homepage and authors here

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+If you wish to contribute to this project, please be sure to
+read/subscribe to the following resources:
+
+ - [Coding Standards](https://github.com/zendframework/zend-coding-standard)
+ - [Forums](https://discourse.zendframework.com/c/contributors)
+ - [Chat](https://zendframework-slack.herokuapp.com)
+ - [Code of Conduct](CODE_OF_CONDUCT.md)
+
+If you are working on new features or refactoring
+[create a proposal](https://github.com/zendframework/zend-developer-tools/issues/new).
```
Comment (webimpress) [inline docs/CONTRIBUTING.md]: `s/zend-developer-tools/ZendDeveloperTools/`

File: composer.json [hunk 3]
```diff
@@ -1,34 +1,20 @@
...
-            "dev-master": "1.1-dev",
-            "dev-develop": "1.2-dev"
-        },
-        "zf": {
-            "module": "ZendDeveloperTools"
-        }
+    "support": {
+        "issues": "https://github.com/zendframework/zend-developer-tools/issues",
+        "source": "https://github.com/zendframework/zend-developer-tools",
+        "rss": "https://github.com/zendframework/zend-developer-tools/releases.atom",
```
Comment (webimpress) [inline composer.json]: `s/zend-developer-tools/ZendDeveloperTools/` and in many other places unless you are going to rename the repo?

Comment (weierophinney) [reply composer.json]: No, this was just from running the zf-maintainer tool. Fixed and pushed now.

Comment (webimpress) [reply composer.json]: Yeh, the tool assumes that repository name is the same as package name defined in `composer.json`.

File: composer.json [hunk 4]
```diff
@@ -63,15 +49,37 @@
...
-        "cs": "phpcs",
+        "cs-check": "phpcs",
         "cs-fix": "phpcbf",
-        "test": "phpunit",
-        "test-coverage": "phpunit --coverage-clover clover.xml"
-    }
+        "test": "phpunit --colors=always",
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+    },
+    "authors": [
```
Comment (webimpress) [inline composer.json]: We can remove authors, as we did in other repos, git log should be enough.

Comment (weierophinney) [reply composer.json]: I left these here as these authors started the project before it came to the zendframework org.

**Pull request merged**